service-tests/wordpress.js

```diff
@@ -2,48 +2,49 @@
 
 const Joi = require('joi');
 const ServiceTester = require('./runner/service-tester');
+const { isMetric } = require('./helpers/validators');
 
 const t = new ServiceTester({ id: 'wordpress', title: 'Wordpress' });
 module.exports = t;
 
 t.create('supported version')
 .get('/v/akismet.json')
 .expectJSONTypes(Joi.object().keys({
-  name: Joi.equal('wordpress'),
+  name: 'wordpress',
   value: Joi.string()
 }));
 
 t.create('plugin version')
 .get('/plugin/v/akismet.json')
 .expectJSONTypes(Joi.object().keys({
-  name: Joi.equal('plugin'),
+  name: 'plugin',
   value: Joi.string()
 }));
 
 t.create('plugin rating')
-.get('/plugin/r/hestia.json')
+.get('/plugin/r/akismet.json')
 .expectJSONTypes(Joi.object().keys({
-  name: Joi.equal('rating'),
+  name: 'rating',
   value: Joi.string()
 }));
 
 t.create('plugin downloads')
-.get('/plugin/dt/hestia.json')
+.get('/plugin/dt/akismet.json')
 .expectJSONTypes(Joi.object().keys({
-  name: Joi.equal('downloads'),
-  value: Joi.string()
+  name: 'downloads',
+  value: isMetric
 }));
 
 t.create('theme rating')
 .get('/theme/r/hestia.json')
 .expectJSONTypes(Joi.object().keys({
-  name: Joi.equal('rating'),
+  name: 'rating',
   value: Joi.string()
```

Comment:
Thanks for catching the bug in the plugin tests!

Do you want to take a minute to improve the rest of these wordpress tests using `Joi.string()` while you're in here? As you pointed out, it masks a lot of obvious problems.

Reply:
I changed the formatting of the badge to use real stars rather than just saying "4.8 stars" for instance. Hope that's okay!

Reply:
It's a little less granular, though I'm fine with it. We could revisit it if folks are unhappy with the change. I wonder if there is a 1/2 star or 1/4 star unicode.

Reply:
[Apparently not](http://www.unicode.org/L2/L2016/16230-half-star-unicode.pdf). We could use something like this, even if it's not ideal: ★★★½

Reply:
Oh! Good thought. I agree it isn't ideal, but using 1/2, 1/4, 3/4 like that seems better than rounding to the nearest star!

Reply:
I'll look into it sometime next week. 😉